docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -34,7 +38,7 @@ Currently, Slack and email notifications are supported.
 - [Enabling and disabling alert policies](#enabling-and-disabling-alert-policies)
 - [Deleting alert policies](#deleting-alert-policies)
 
-### Creating alert policies
+### Creating run-based alert policies
```

Comment:
I can't comment on unchanged lines, but the anchor on line 36 has to be updated since the heading here has changed.

Reply:
Additionally, it feels a bit odd to call this section "run-based alert policies" when runs haven't been introduced / described in the guide. Perhaps add something about (job) runs in the **Understanding alert policies** section?